orchestrator/db/models.py

```diff
@@ -624,6 +626,7 @@ class SubscriptionMetadataTable(BaseModel):
         index=True,
     )
     metadata_ = mapped_column("metadata", pg.JSONB(), nullable=False)  # type: ignore
+    version = mapped_column(Integer, nullable=False, server_default="1")
```

Comment:
accidentally left this in. metadata does not need version control, there is no api endpoint to update it. we use a workflow to update it.
```suggestion
```